### PR TITLE
2022-07-31 release notes

### PR DESCRIPTION
**Pull request by justindavies**

### CODE REVIEW

**Comment by olsenme** [comment]
lgtm

**Review by miwithro — CHANGES_REQUESTED** [review]
KMS is also GA!!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+
+### Release notes
+
+* Features
+  * [Dedicated Host Support](https://docs.microsoft.com/en-us/azure/aks/use-azure-dedicated-hosts) is now generally available.
+  * [Confidential Virtual Machines](https://docs.microsoft.com/en-us/azure/aks/use-cvm) is now in Public Preview.
+* Behavioral Changes
+  * Use QuotaExceeded error code instead of OperationNotAllowed when receiving quota exceed errors from ARM
+
+* Bug Fixes
```
Comment (palma21) [inline CHANGELOG.md:30]: Extra line

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Konnectivity rollout is finished in global and started in Sovereign (China, USGov).
+* Docker will no longer be supported as a container runtime on Windows after September 1, 2022. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+
+### Release notes
+
+* Features
+  * [Dedicated Host Support](https://docs.microsoft.com/en-us/azure/aks/use-azure-dedicated-hosts) is now generally available.
+  * [Confidential Virtual Machines](https://docs.microsoft.com/en-us/azure/aks/use-cvm) is now in Public Preview.
```
Comment (palma21) [inline CHANGELOG.md:26]: Missing KMS

**Review by palma21 — APPROVED** [review]

**Pull request merged**